src/styles/blocks/card/blockscard/cardBuy.scss

```diff
@@ -0,0 +1,23 @@
+.card__buy {
+  box-sizing: border-box;
+  height: 40px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  text-align: center;
+  background: $link-bg-color;
+  color: $link-text-color;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  border: 1px solid $link-bg-color;
+
+  &:hover {
+    background-color: $link-text-color;
+    color: $link-bg-color;
+    border-color: $link-bg-color;
+    line-height: 39px;
```

Comment:
The `line-height` property is not usually changed on hover. This property is used to set the amount of space used by a line of text. Changing it on hover can lead to unexpected visual results. It's better to use padding or margin to adjust element's dimensions.